spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/MappingSpannerConverter.java

```diff
@@ -246,8 +265,13 @@ public void write(Object source, WriteBuilder sink, Set<String> includeColumns)
 		this.writeConverter.write(source, sink, includeColumns);
 	}
 
+	@Override
+	public <R> R read(Class<R> type, Struct source, Set<String> includeColumns, boolean allowMissingColumns) {
+		return this.entityReader.read(type, source, includeColumns, allowMissingColumns);
+	}
+
 	@Override
 	public <R> R read(Class<R> type, Struct source) {
```

Comment:
Does this overriding method have any effect? It looks like the `default` version defined calls the other more-fully defined `read` method, which is exactly what we're doing here because `this.entityReader.read` is just `default` version.